modules/primer-marketing/package.json

```diff
@@ -1,11 +1,12 @@
 {
-  "version": "6.0.2",
+  "version": "6.0.3",
```

Comment:
Should this be a minor bump seeing as there are minor changes to packages within it?